internal/editor/editor.go

```diff
@@ -47,8 +59,17 @@ func Launch(repo *git.Repo, config Config) (string, error) {
 		return "", err
 	}
 
-	// launch the editor as a subprocess
-	cmd := exec.Command(config.Command, tmp.Name())
+	// Launch the editor as a subprocess.
+	// We interpret the command with shell syntax to allow users to specify
+	// both flags and use editor executables with spaces.
+	// e.g., EDITOR="'/path/with spaces/editor'" or
+	// EDITOR="code --wait" work.
+	args, err := shellquote.Split(config.Command)
+	if err != nil {
+		return "", errors.Wrapf(err, "invalid editor command: %q", config.Command)
+	}
+	args = append(args, tmp.Name())
+	cmd := exec.Command(args[0], args[1:]...)
```

Comment:
you are not using `tmp.Name()` anymore, is that intentional?

Reply:
Yeah, now it's inside `args` (via the `args = append(args, ...)` the line above). Go isn't quite as ergonomic as Python about when you can splat things (I think to get more control over when it does allocations and whatnot).

```go
// The function is defined as
func Command(name string, arg ...string) *Cmd

// Illegal because the first argument can't be given as part of the splat
cmd := exec.Command(args..., "foo")

// Illegal because you can't add things after a splat
cmd := exec.Command(args[0], args[1:]..., "foo")

// Legal because there's no splat and Go can coalesce the varags into a single array
cmd := exec.Command("foo", "bar", "spam", "eggs")
// equivalent to
cmd := exec.Command("foo", []string{"bar", "spam", "eggs"}...)
```